### PR TITLE
Add Example For nRF52

### PR DESCRIPTION
**Pull request by felix-gohla**

### CODE REVIEW

File: examples/nrf52/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+ARG SEGGER_JLINK_VERSION=V766g
+
+# SEGGER JLink Tools (closed source >.<)
+RUN cd /tmp && \
+    if [ "$TARGETARCH" = "amd64" ]; then ARCHITECTURE=x86_64; elif [ "$TARGETARCH" = "arm64" ]; then ARCHITECTURE=arm64; else ARCHITECTURE=x86_64; fi && \
+    curl -X POST -d "accept_license_agreement=accepted" https://www.segger.com/downloads/jlink/JLink_Linux_${SEGGER_JLINK_VERSION}_${ARCHITECTURE}.deb --output JLink_Linux_${SEGGER_JLINK_VERSION}_${ARCHITECTURE}.deb && \
+    dpkg -i JLink_Linux_${SEGGER_JLINK_VERSION}_${ARCHITECTURE}.deb && \
+    rm JLink_Linux_${SEGGER_JLINK_VERSION}_${ARCHITECTURE}.deb
+
```
Comment (sirkrypt0) [inline examples/nrf52/Dockerfile:92]: On my system, the `debug` command fails with `FileNotFoundError: [Errno 2] No such file or directory: '/opt/SEGGER/JLink_V766g/jlinkarm.so'`, which might have to do with the installer missing to create the symlink.

Quick fix would be

```suggestion
RUN if [ "$TARGETARCH" = "amd64" ]; then ln -s /opt/SEGGER/JLink/libjlinkarm.so /opt/SEGGER/JLink/jlinkarm.so; fi
```

**Review by sirkrypt0 — CHANGES_REQUESTED** [review]

File: examples/nrf52/Dockerfile (hunk 1 above)
Comment (felix-gohla) [reply examples/nrf52/Dockerfile:93]: Better now? 😊

**Review by frcroth — APPROVED** [review]
Works for me

File: examples/nrf52/Dockerfile (hunk 1 above)
Comment (sirkrypt0) [reply examples/nrf52/Dockerfile:93]: Works for me :+1:

**Review by sirkrypt0 — APPROVED** [review]

**Pull request merged**